docs/hyphae.md

```diff
@@ -0,0 +1,35 @@
+# Hyphae Setup
+
+Hyphae is a filament, or bridge, between Myceli and Kubo. It allows provides a pathway for the IPFS blocks inside of Myceli to flow into Kubo, and from there potentially into the broader public IPFS network.
+
+## Running Hyphae
+
+After building from source, or downloading a binary, `hyphae` can be run with no additional arguments:
+
+    $ hyphae
+
+Starting hyphae with no config file will run with a few default settings: 
+- Looking for `myceli` at `127.0.0.1:8080`
+- Looking for `kubo` at `127.0.0.1:5001`
+- Syncing data every 10 seconds
+
+Every ten seconds, `hyphae` will query `myceli` for it's available blocks, query `kubo` for it's local refs, and transfer over any blocks which exist in `myceli` and not in `kubo`.
```

Comment:
For a future refactor, is it possible for myceli to push changes to hypahe to avoid the polling sync?

Reply:
Maybe hypahe subscribes to myceli for any "received full blocks" events?